### PR TITLE
re-work lite clients into 'Browsers & Mobile'

### LINKED ISSUE #31: Interest in an 'IPFS in Browsers' talk/workshop/session?
I'd be interested in attending and perhaps being part of a focused 'IPFS in Browsers' discussion/workshop, talking about all the things related to making IPFS read and write access feasible in browsers, covering the gamut of access and usage patterns/apis via gateways, js-ipfs, js-ipfs-fetch, IPFS Companion, extensions vs regular pages, services like web3.storage, native support in Brave, Agregore, mobile, etc...
(Perhaps working out some of the issues @RangerMauve has begun to summarize here webrecorder/archiveweb.page#85)

Wanted to gauge interest before opening a PR @lidel @autonome @RangerMauve

### PR DESCRIPTION
**Pull request by b5**
Hey all, I've restructured the "Lite Clients" track to include browsers and provide a goal for this half-day workshop. I'd love your feedback!

@RangerMauve & @matheus23, I'm asking for both of you (or someone from your org) to give lightning talks to help support this goal of moving the ball forward on IPFS in weird places. I'll wait for both of your feedback & approval before merging.

I'm also seeking feedback from folks I'd love to have attend:
@achingbrain
@autonome
@ikreymer
@lidel
@dignifiedquire

My first request for feedback: who else should we be trying to get to attend?

closes #31

### CODE REVIEW

**Comment by matheus23** [comment]
I can commit to preparing a lightning talk :+1:

**Comment by autonome** [comment]
* add a "DIY content routing" session using Reframe for a "lite client" approach to DHT alternatives

File: events/d_3_browsers_and_mobile.toml [hunk 1]
```diff
@@ -66,79 +68,61 @@ Carving a path forward for an IPFS lite client story would open up massive categ
...
 
 [[timeslots]]
-startTime="11:45"
-speakers=[]
-title="Open Slot"
-description=""
+startTime="13:25"
+speakers=["@RangerMauve?"]
+title="Agregore `fetch()`, a multi-protocol abstraction"
+description="this talk is unconfirmed"
```
Comment (RangerMauve) [inline events/d_3_browsers_and_mobile.toml:86]:
```suggestion
description="This talk will look at Peer to Peer URLs and how they can be used in tandem with HTTP-like APIs within webapps using custom protocol handlers."
```

**Review by RangerMauve — CHANGES_REQUESTED** [review]
Yeah, I'd love to talk about Agregore's approach to URLs and `fetch()`. 😁 One thing I'd like to focus is some existing `ipfs://` and `ipns://` URL protocol handlers in stuff like Brave and Capyloon and how they can become more powerful with the addition of `PUT/POST/DELETE` verbs.

**Pull request merged**